src/managed/OpenLiveWriter.Localization/StringId.cs

```diff
@@ -5169,7 +5169,11 @@ public enum StringId
         /// <summary>
         /// There was an unexpected error while uploading the video.
         /// </summary>
-        YouTubeVideoError
+        YouTubeVideoError,
+        /// <summary>
+        /// percent
+        /// </summary>
+        percent
```

Comment:
For consistency, can we please make this an upper case P as well as a proper description in the summary tags.